lxd/backup/backup_instance.go

```diff
@@ -47,6 +49,11 @@ func (b *InstanceBackup) InstanceOnly() bool {
 	return b.instanceOnly
 }
 
+// Instance returns the instance to be backed up
```

Comment:
Tiny nit but these days we like our comments to end with a period. This wasn't always the case so it's still a bit of a mixed mess but new ones we prefer to have it included.